pkg/scalers/rabbitmq_scaler.go

```diff
@@ -114,6 +121,24 @@ func parseRabbitMQMetadata(resolvedEnv, metadata, authParams map[string]string)
 		}
 	}
 
+	if strings.HasPrefix(meta.host, amqps) {
+		if val, ok := authParams["ca"]; ok {
+			meta.ca = val
+		} else {
+			return nil, fmt.Errorf("no ca given")
+		}
+		if val, ok := authParams["cert"]; ok {
+			meta.cert = val
+		} else {
+			return nil, fmt.Errorf("no cert given")
+		}
+		if val, ok := authParams["key"]; ok {
+			meta.key = val
+		} else {
+			return nil, fmt.Errorf("no key given")
```

Comment:
from the usage below, I'm assuming those are expected to have file paths, not actual certificate values, right? I think you would want to use `metadata` map instead of the `authParams` map, as the latter contains values resolved from a `TriggerAuthentication` CRD if present for a particular trigger.

Reply:
also a bit of nitpicking, but perhaps `rabbitmq host is using amqps://, but no key/cert/ca provided` would be a bit more descriptive error message.

Reply:
> also a bit of nitpicking, but perhaps `rabbitmq host is using amqps://, but no key/cert/ca provided` would be a bit more descriptive error message.

Done